src/login/logind.conf.in

```diff
@@ -50,3 +50,4 @@
 #InhibitorsMax=8192
 #SessionsMax=8192
 #StopIdleSessionSec=infinity
+#MaintenanceWindow=
```

Comment:
i am kinda tempted to set a time here by default. What do Chromebooks use for this? any idea?

I mean, just setting a time here has no effect on its own as long as noone schedules a shutdown without specifying an explicit time, hence it should be safe to just default to something reasonable here? maybe "3:00"?

Reply:
(if we default to 3:00, then this should be primarily encoded in the C code, but also here in the default config file, for illustrative purposes)

Reply:
not sure that's such a great idea for standard, non-transactional rw systems. Also would break the compatibility with the standard 1 minute shutdown -r timer.